hack/after-build/update-generated-docs.sh

```diff
@@ -39,8 +39,10 @@ shopt -u dotglob
 
 kube::util::gen-analytics "${KUBE_ROOT}"
 
+KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
```

Comment:
Might be, certainly for cherry-pick scripts.

On Tue, Apr 26, 2016 at 9:04 PM, David McMahon <notifications@github.com>
wrote:

> In hack/after-build/update-generated-docs.sh
> <https://github.com/kubernetes/kubernetes/pull/24666#discussion_r61189988>
> :
>
> > @@ -39,8 +39,10 @@ shopt -u dotglob
> >
> >  kube::util::gen-analytics "${KUBE_ROOT}"
> >
> > +KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
>
> Is there a better place under hack/lib to set this once rather than in
> both *-generated-docs.sh scripts?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24666/files/9f2da6b2b33d3f6fa4047e55b4d835d504612fd5#r61189988>
>